src/applications/hca/components/FormFields/VaMedicalCenter.jsx

```diff
@@ -110,14 +110,14 @@ const VaMedicalCenter = props => {
     [localData, onChange],
   );
 
-  const localDataFacilityState = localData['view:facilityState'];
   const previousFacilityState = usePrevious(localData['view:facilityState']);
 
   // fetch, map and set our list of facilities based on the state selection
   useEffect(
     () => {
       const { 'view:facilityState': facilityState } = localData;
-      if (facilityState) {
+
+      if (facilityState && facilityState !== previousFacilityState) {
```

Comment:
🤔 not sure if it's worth it to pull this predicate into a reusable function or constant? it's only two instances of it, but I feel like there's a risk of the two conditionals drifting apart?

```suggestion
      const selectedStateHasChanged = facilityState && facilityState !== previousFacilityState;
      if (selectedStateHasChanged) {
```
?

Reply:
I ended up moving this to a variable to make it more readable 👍